src/flux/transition.coffee

```diff
@@ -23,9 +25,9 @@ TransitionStore = flux.createStore
   actions: _.values(TransitionActions)
   _local: []
 
-  load: ({path, type}) ->
-    type ?= 'push'
-    @_local.push(path) if type is 'push'
+  load: (change, router) ->
+    change.type ?= 'push'
```

Comment:
hehehehe, this is a similar comment that @philschatz  made when I made this original PR -- this will change the `change` object. just to be more safe, could we do, `{type, path}`and `type ?= 'push'`?